extensions/amp-auto-ads/0.1/placement.js

```diff
@@ -315,6 +319,18 @@ export class Placement {
       attributes
     );
   }
+
+  /**
+   * Estimate if the viewport has a narrow layout.
+   * @param {!Element} element
+   * @return {boolean}
+   * @private
+   */
+  isLayoutViewportNarrow_(element) {
+    const viewportSize = Services.viewportForDoc(element).getSize();
+
+    return viewportSize.width < 488;
```

Comment:
Could you add a comment explaining the reasoning behind 488.

Reply:
Thanks. Added the comment. This is to align with our threshold for non-AMP traffics. You can code search isLayoutViewportNarrow in responsive_util.js for the internal reference.